app/styles/app.scss

```diff
@@ -212,4 +204,72 @@ body {
     line-height: 16px;
     color: #607787;
  }
- 
\ No newline at end of file
+
+ .container2 {
+   display: flex;
+   flex-direction:row;
+ }
+
+ .dropdown-container {
+   display: flex;
+   margin-left:80px;
+   flex-direction: row;
+}
+
+.dropdown-text{
+   margin:10px;
+   font-family: 'SF Pro Text';
+   font-style: normal;
+   font-weight: 400;
+   font-size: 14px;
+   line-height: 20px;
+   letter-spacing: -0.08px;
+   color: #7B8E9F;
+ }
+
+ .dropdown-toggle::after {
+   content: none;
+}
+
+ .vector {
+   margin:10px;
+ }
+
+ .vector:hover {
+   width:9px;
+   height:9px;
+ }
+
+ .dropdown-toggle {
+   display: flex;
+   flex-direction: row;
+   font-family: 'SF Pro Text';
+   font-style: normal;
+   font-weight: 600;
+   font-size: 14px;
+   line-height: 20px;
+   letter-spacing: -0.08px;
+   color: #12344D;
+   margin:10px;
+ }
+
+ a:hover {
+   text-decoration: none;
+ }
+
+ .item-lists {
+   font-family: 'SF Pro Text';
```

Comment:
We can avoid repeating css properties by setting them in common parent class or element. And, font-family, font-weight, and size can be defined in a common variable file and can be used.